plaid/client.py

```diff
@@ -219,14 +219,16 @@ def _step(self, credentials, url, method, account_type, mfa, options=None):
 
     @store_access_token
     @inject_credentials
-    def _update(self, credentials, url, login):
+    def _update(self, credentials, url, login, options=None):
         '''
-        Similar to _add, save HTTP method, inclusion of access token,
-        and absence of options / account_type
+        Similar to _add, save HTTP method, inclusion of access token.
         '''
+        data = dict(login, **credentials)
+        data['options'] = json.dumps(options if options is not None else {})
```

Comment:
Updated, but please note that this doesn't allow for Falsey values (options=False) and that it is contrary to options handling throughout the rest of the module. I originally maintained consistency with everything else because I was unsure whether or not options=False or similar values would ever be valid for Plaid's API.